index.js

```diff
@@ -72,6 +73,7 @@ module.exports = {
         }
       },
       rules: {
+        'react/dom-elements-style-is-object': 2,
```

Comment:
I think we want to wait to enable this in the recommended configuration until the next major version bump, since doing so may end up breaking people's builds.